src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Label/PaymentState/orderPaymentState.html.twig

```diff
@@ -0,0 +1,16 @@
+{%
+    set viewOptions = {
+        awaiting_payment: {icon: "clock icon", color: "grey"},
+        partially_authorized: {icon: "check icon", color: "yellow"},
+        authorized: {icon: "check icon", color: "orange"},
+        partially_paid: {icon: "adjust icon", color: "olive"},
+        cancelled: {icon: "ban icon", color: "red"},
+        paid: {icon: "check icon", color: "green"},
+        partially_refunded: {icon: "reply icon", color: "violet"},
+        refunded: {icon: "reply all icon", color: "purple"},
```

Comment:
There should be always a space after `{` and before `}`

Reply:
And if we put the `icon` class to `i` element in html, it would not have to be repeated everywhere :)

Reply:
Moreover, the alphabetical order is not kept xD

Reply:
This refers to this and the next file

Reply:
This one https://github.com/Sylius/Sylius/pull/10233#discussion_r271629093 is still relevant